setup.cfg

```diff
@@ -4,5 +4,8 @@ python-tag = py3
 [doc8]
 max-line-length = 99
 
+[flake8]
+extend-exclude = venv
```

Comment:
This is not necessary for this PR, but helps with local dev, as you can just run `flake8` and it checks everything